server.js

```diff
@@ -1,21 +1,28 @@
 require('dotenv').config();
 
 const { createApp } = require("./app");
-const dataSource = require("./api/models/dataSource");
+const { dataSource } = require("./api/models/dataSource");
 
 const startServer = async () => {
     const app = createApp();
     const PORT = process.env.PORT;
 
-        await dataSource.initialize()
+        await dataSource
+        .initialize()
+        .then(() =>{
+            console.log("Data source has benn initialized");
+        })
+        .catch((error) => {
+            console.log("Error during Data Source initialization", error);
+        });
 
         app.listen(PORT, () => {
-            console.log('Listening to request om PORT')
+            console.log('Listening to request on PORT')
```

Comment:
PORT 확인이 필요한 경우 다음과 같이 `template-literal` 사용하여 작성하는 것이 좋을 것 같습니다.

```suggestion
            console.log(`Listening to request on ${PORT}`)
```